assemblies/assembly-business-optimizer-vrp.adoc

```diff
@@ -1,5 +1,5 @@
 [id='assembly-business-optimizer-vrp']
-= Deploying and using the vehicle route planning starter application for {PLANNER}
+= Deploying and using the {PLANNER} vehicle route planning start application
```

Comment:
"starter application" is used in the rest of the documentation.

Reply:
Fixed